client/src/js/TinyMCE_ssmedia.js

```diff
@@ -94,7 +108,41 @@
 
             const img = `<img src="${secureUrl}" />`;
             this.editor.execCommand('mceInsertContent', false, img);
-        }
+        },
+
+        /**
+         * Inserts the data into the editor as a link
+         *
+         * @param {*} response
+         */
+        linkHandler(response) {
+            const asset = response.assets[0];
+
+            let secureUrl;
+            if (asset.derived && asset.derived.length > 0) {
```

Comment:
For the sake of readability, can you put a newline before and after if/else statements?

Reply:
Done